pyaedt/icepak.py

```diff
@@ -155,7 +155,7 @@ def assign_openings(self, air_faces):
         bound = BoundaryObject(self, boundary_name, props, 'Opening')
         if bound.create():
             self.boundaries.append(bound)
-            self._messenger.add_info_message("Opening assigned.")
+            self._messenger.add_info_message("Opening Assigned")
```

Comment:
```suggestion
            self._messenger.add_info_message("Opening assigned.")
```
@PipKat you can leave your changes to make the message `"Opening assigned."`, that is not a problem. The docstring example just needs to be updated to reflect this. So on line 142, you can update it to be `pyaedt Info: Opening assigned.`